src/shared/components/query/GeneSetSelector.tsx

```diff
@@ -40,16 +41,21 @@ export default class GeneSetSelector extends QueryStoreComponent<GeneSetSelector
 
 	@computed get geneListOptions()
 	{
-		return [
-			{
-				label: 'User-defined List',
-				value: ''
-			},
-			...gene_lists.map(item => ({
-				label: `${item.id} (${item.genes.length} genes)`,
-				value: item.genes.join(' ')
-			}))
-		];
+	    let geneList: {"id": string, "genes": string[]}[] = gene_lists;
+	    if (AppConfig. querySetsOfGenes) {
```

Comment:
There is an extra white space after `.`, not sure if it effects the execution.

Reply:
hmm, that shouldn't be here (although I tested it and it was working). I will fix it.